blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/AstyanaxReader.java

```diff
@@ -47,8 +48,8 @@
 import org.slf4j.LoggerFactory;
 
 import java.io.IOException;
-import java.sql.Connection;
 import java.util.*;
+import java.util.concurrent.*;
 
 public class AstyanaxReader extends AstyanaxIO {
```

Comment:
Note that changes for this class are copied from the "Adding queries for enums" PR 527:  https://github.com/rackerlabs/blueflood/pull/527, because I need the functionalities added in that story for my own use (i.e. the getColumnsFromEnumCF() and getEnumValueFromHash() methods).  
Once that PR is merged in, I'll rebase and use that PR copy of AstyanaxReader instead.